charley_tran/week_01/trainsHW.html

```diff
@@ -0,0 +1,107 @@
+<html>
+<script>
+//planTrip('N', 'Times Square', '6', '33rd')
+// console.log() shows output similar to this:
+// "Your must travel through the following stops on the N line: 34th, 28th, 23rd, Union Square."
+// "Change at Union Square."
+// "Your journey continues through the following stops: 23rd, 28th, 33rd."
+// "7 stops in total."
+
+
+// improvements I could have made. make the counting of the stops based on array length of the stations rather than between stop a and stop b 
+// is it possible to declare lineString1 and lineString2 locally into the planTrip function so it resets the array after each trip (no carryover of global data??) EDIT : YES it does work, will have to do it for count stops as well
+
+//=====================
+
+var lineN = ['times square', '34th', '28th.n', '23rd.n', 'union square', '8th.n'];
+var lineL = ['8th.l', '6th', 'union square', '3rd', '1st'];
+var line6 = ['grand central', '33rd', '28th.6', '23rd.6', 'union square', 'astor place'];
+
+// var nExchange = lineN[4];
+// var lExchange = lineL[2];
+// var Exchange6 = line6[4];
+
+var lineStations = [];      //line of stations
+// var lineString1 = [];      //line string 1 and line string 2 stores the strings from stop a to union and stop b to union
+// var lineString2 = [];
+var stringLineStation;    //converison of array to strings
+
```

Comment:
To be honest I fluked this homework. My understanding of objects in objects was still limited. If I was to redo it I'd write the stations as an object, with 3 objects containing arrays. 
Also my string manipalation is better now; so I wouldn't have to add the values into an array but straight into an object.
Made my job alot harder but I'm sure I'd be able to simplify it down now.